gehomesdk/erd/converters/laundry/remote_control_converter.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from gehomesdk.erd.converters.abstract import ErdReadWriteConverter
+from gehomesdk.erd.converters.primitives import *
+
+_LOGGER = logging.getLogger(__name__)
+
+class LaundryRemotePowerControlConverter(ErdReadWriteConverter[int]):
```

Comment:
Should probably use ErdIntConverter instead of a custom converter unless I'm misunderstanding this

Reply:
I wasn't sure how to encode as a 1 byte or 2 byte value using that class. 

Eg 2038 is a 2 byte (0000) but 2040 is a 1 byte (00) values.  Is there a way to specify the byte length?

Reply:
yep, if you specify the "length" parameter in the constructor, you can change that.  so, it would be something like ErdIntConverter(length=2) in the erd_configuration.py file.

Reply:
Interestingly, the encoder had a bug and was ignoring length.  It's fixed once you pull down the newest changes.

Reply:
thank you, should be good to go now.